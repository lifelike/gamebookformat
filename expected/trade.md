# Gamebook #


## Introduction ##
 Testing trade of things. You can buy things using Gold of which you start with 100. You also have an Inventory. At start you carry a **sword**
. 

Turn to 1 to begin.
<a name="section1">

## 1 ##
 You can buy a **shield**
 here for 50 Gold or a beautiful **gold ring**
 for only 75. There is also a **chain mail**
 costing 150. You can pick up some **spiderweb**
 for free. You can go to [4](#section4)
 at a cost of 50. You can go on to the end at [2](#section2)
. 
<a name="section2">

## 2 ##
 It ends here. There is a **crown**
 here that you can trade for some spiderweb. 
<a name="section3">

## 3 ##
 This is somewhere else. Congratulations you have found 100 gold! Go back to [1](#section1)
. 
<a name="section4">

## 4 ##
 Now you have 50 less gold than you just had. You can go back to [1](#section1)
. 

