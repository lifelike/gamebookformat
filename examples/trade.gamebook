= Introduction
Testing trade of things. You can buy things using
[count gold]Gold[/count] of which you start with [set gold]100[/set].
You also have an [collect item]Inventory[/item]. At start you carry
a [add item]sword[/add].

* 1 start
You can buy a [found item]shield[/found] here for [cost gold]50[/cost] Gold
or a beautiful [found item]gold ring[/found] for only [cost gold]75[/cost].
There is also a [found item]chain mail[/found] costing [cost gold]150[/cost].
You can pick up some [found item]spiderweb[/found] for free.
You can go to [[other]] at a cost of [cost gold]50[/cost].
You can go on to the end at [[theend]].

* other :HTML:TWINE2:
Hopefully you now have 50 less gold than you just had.
You can go back to [[start]].
You can also go to [[someotherplace]] but an angry troll
is blocking your path and will only allow you to go there
if you give it a [trade item]gold ring[/trade].

* other :TXT:JSON:TEX:DOT:RTF:MD:
Now you have 50 less gold than you just had.
You can go back to [[start]].

* someotherplace
This is somewhere else. Congratulations you have found [inc gold]100[/inc] gold!
Go back to [[start]].

* theend
It ends here. There is a [found item]crown[/found] here that you can trade for
some [trade item]spiderweb[/trade].

