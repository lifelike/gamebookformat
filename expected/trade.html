<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Gamebook</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'trade';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="introsection">
<div class="introsectionheading">Introduction</div>
<div class="introsectionbody">
 Testing trade of things. You can buy things using <span class="count counterChange" data-type="gold"
    data-name="Gold">Gold</span> of which you start with <span class="set counterChange" data-type="gold"
      data-amount="100">100</span>. You also have an <span class="collect" data-type="item"
    data-name="Inventory">Inventory</span>. At start you carry a <span class="add" data-type="item"
      data-what="sword">sword</span>. 
</div>
</div>
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1"
     >
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 You can buy a <span class="found enabledlink" data-type="item"
      data-what="shield"
>shield</span>
 here for <span class="cost" data-type="gold"
      data-amount="50"
>50</span>
 Gold or a beautiful <span class="found enabledlink" data-type="item"
      data-what="gold ring"
>gold ring</span>
 for only <span class="cost" data-type="gold"
      data-amount="75"
>75</span>
. There is also a <span class="found enabledlink" data-type="item"
      data-what="chain mail"
>chain mail</span>
 costing <span class="cost" data-type="gold"
      data-amount="150"
>150</span>
. You can pick up some <span class="found enabledlink" data-type="item"
      data-what="spiderweb"
>spiderweb</span>
 for free. You can go to <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a> at a cost of <span class="cost" data-type="gold"
      data-amount="50"
>50</span>
. You can go on to the end at <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script>
<div class="section" id="section2"
     >
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 It ends here. There is a <span class="found enabledlink" data-type="item"
      data-what="crown"
>crown</span>
 here that you can trade for some <span class="trade" data-type="item"
      data-what="spiderweb"
>spiderweb</span>
. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script>
<div class="section" id="section3"
     >
<div class="sectionnumber" id="para3">3</div>
<div class="sectiontext">
 This is somewhere else. Congratulations you have found <span class="inc counterChange" data-type="gold"
      data-amount="100">100</span> gold! Go back to <a class="sectionref enabledlink" data-ref="1"
href="#section1">1</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(3, document.getElementById('section3'));
}
</script>
<div class="section" id="section4"
     >
<div class="sectionnumber" id="para4">4</div>
<div class="sectiontext">
 Hopefully you now have 50 less gold than you just had. You can go back to <a class="sectionref enabledlink" data-ref="1"
href="#section1">1</a>. You can also go to <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> but an angry troll is blocking your path and will only allow you to go there if you give it a <span class="trade" data-type="item"
      data-what="gold ring"
>gold ring</span>
. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(4, document.getElementById('section4'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
